edx_api/email_settings.py

```diff
@@ -0,0 +1,60 @@
+"""
+API client for interacting with change email settings
+"""
+import logging
+from urllib import parse
+
+log = logging.getLogger(__name__)
+
+
+class ChangeEmailSettings(object):
```

Comment:
I have a few suggestions regarding this so I'd list them down in a single place like here:

1) Like other API's can we move this to a separate Django app like a structured directory? Like it's done [here](https://github.com/mitodl/edx-api-client/tree/master/edx_api/enrollments) and [here](https://github.com/mitodl/edx-api-client/tree/master/edx_api/course_detail). This will make it cleaner and more standardized.

2) I think this name is too specific and it would be a good idea to at least generalize it in the sense that it represents `Email Settings` overall. Could you please name this class `EmailSettings`? This will also keep things consistent with the existing overall client structure.

Reply:
@arslanashraf7 regarding your first point, we have don't big enough modules for it that's why I preferred to keep it here as we have done in the case of CCX and client modules.